scss/civicrm/contact/pages/_export.scss

```diff
@@ -22,8 +25,19 @@
     margin-left: 0 !important;
   }
 
-  #export-type {
-    padding: 0 20px;
+  .crm-section {
+    margin-bottom: 0;
+    padding-bottom: 20px;
+
+    &.crm-export-mapping-section {
+      padding-bottom: 0;
+    }
+  }
+
+  .crm-section {
```

Comment:
Can this styles be included at https://github.com/civicrm/org.civicrm.shoreditch/pull/101/files#diff-3a3f482f6902e2b4a41682a5288fb85bR28 ?

Reply:
Line 28 same file